PRINCIPLES.md

```diff
@@ -158,3 +166,46 @@ _Principle: We help high-quality high-velocity cloud native open source projects
 We want to be able to say that CNCF is a net positive for big & small projects. Doing so requires more coordination with project leads. 
 
 Project needs may include test automation and CI, cloud resources to test on, clear documentation, per-project marketing & evangelism, roadmaps for interop, and advice from experts on governance and scalability. And we need to make sure project contributors see what value they are getting & are not afraid to ask for help!
+
+
+## Technical Leadership Principles
+
+### Purpose
+
+The Technical Oversight Committee has adopted these leadership principles to help guide the ecosystem in growth and maturity. These principles define the expectations of TOC members, project maintainers, TAG Chairs and Leads, chairs and leads of any group or committee, and other persons serving in any leadership role in the CNCF community.
+
+### Principles
+
+Our goal with these is to foster a more inclusive and welcoming community for all, and we expect all leaders in the CNCF community  to participate in accordance with these principles. If a leader is not acting in accordance with these principles, community members should first address it with the other leaders of the project, TAG, committee, group, or body. Project and TAG governance documentation must explain procedures (including who is responsible) for response and resolution if these principles are not being followed. In general, if a leader is not following these principles, there should be a good faith effort to educate, inform, and provide the leader with an opportunity to improve before taking further action such as suspending or removing them from their leadership role. If the behavior that violates these principles persists, or if the other leaders of the applicable project, committee, or group do not take corrective action in response to a reported violation of these principles, projects should escalate the matter to the TOC.
+
+In addition to adhering to these principles, all leaders must adhere to the CNCF Code of Conduct. 
+
+#### Be welcoming
+
+Make a conscious effort to be welcoming in your interactions. If someone is new, express sincere interest in what brought them to the group. Check in with existing contributors to ensure they continue to feel welcome. Create space for new contributors and existing contributors to be curious and engaged with the work that we do in cloud native by building on common interests and outcomes. 
+
+#### Provide feedback and appreciation appropriately
+Provide praise, gratitude, appreciation and encouragement in public. Look for opportunities to say “thank you” and recognize people when they are being helpful or making contributions. Be specific in what was done and describe the impact it had or continues to have.
```

Comment:
Request to add something around "Make sure credit is given correctly, either in github references, or authorship locations".  Something around that?